src/services/youtube.service.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const KEY = 'AIzaSyC3gRHp_IaKRrA7SOnafALyuhyt815KVEM'
```

Comment:
It is not a good idea to have your API keys publicly available.  I recommend you look into [Adding Custom Environment Variables](https://create-react-app.dev/docs/adding-custom-environment-variables/) in React to achieve the goal of having API keys locally and on the deployed server but not on Github

Reply:
Definitely, it's not good idea expose API keys, I'm going to fix it.
Thank so much for your time